pom.xml

```diff
@@ -1658,6 +1658,11 @@
                 <artifactId>mysql-connector-java</artifactId>
                 <version>${mysql.connector.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.postgresql</groupId>
+                <artifactId>postgresql</artifactId>
+                <version>42.2.25</version>
```

Comment:
Maybe we should introduce a variable and reuse it here and there: https://github.com/hazelcast/hazelcast/blob/380baa88306dd36e3527c00978f97d3351b3c2c9/hazelcast/pom.xml#L727